packages/server/modules/core/migrations/20250123162109_required_indices.ts

```diff
@@ -0,0 +1,85 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('stream_commits', (table) => {
+    table.index('commitId')
+  })
+
+  await knex.schema.alterTable('branch_commits', (table) => {
+    table.index('commitId')
+  })
+
+  await knex.schema.alterTable('refresh_tokens', (table) => {
+    table.index('userId')
+  })
+
+  await knex.schema.alterTable('commits', (table) => {
+    table.index('author')
+    table.index('sourceApplication')
+    table.index('referencedObject')
+    table.index('createdAt')
+  })
+
+  await knex.schema.alterTable('file_uploads', (table) => {
```

Comment:
can you provide references to how these other indexes that don't have comments are used?